clients/java/signalr/src/main/java/com/microsoft/signalr/HubException.java

```diff
@@ -4,6 +4,8 @@
 package com.microsoft.signalr;
 
 public class HubException extends Exception {
+    private static final long serialVersionUID = -572019264269821519L;
```

Comment:
Why this number? Why not just `1`? AFAICT this just has to be a number that is incremented whenever we make a change that would affect serialization.